roster2/tool/src/java/org/sakaiproject/roster/impl/SakaiProxyImpl.java

```diff
@@ -297,10 +297,15 @@ public Boolean getViewEmail(String siteId) {
     }
 
     @Override
-    public Boolean  getViewUserDisplayId() {
+    public Boolean getViewUserDisplayId() {
```

Comment:
Is this method without the parameter even still used? I'm having a problem checking out this PR so can't check but I'm not seeing it anymore.

Reply:
I think this is *fine* to leave in if the NPE is checked in the other method?